### PR TITLE
User can view their entries

### PR DESCRIPTION
**Pull request by danielycho**
function 2

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -57,3 +57,56 @@ textarea {
...
+.nav-style {
+  padding-left: 2rem;
+  text-decoration: none;
+  font-size: 1.3rem;
+  background-color: #562b81;
+  border: none;
+}
+
+.align {
+  margin-left: 25rem;
```
Comment (codingcodymiller) [inline css/styles.css]: WOAH this is a huge margin. Why is this necessary? There is almost certainly a better way to accomplish this, likely with `justify-content` or `text-align`

File: css/styles.css [hunk 2]
```diff
@@ -57,3 +57,56 @@ textarea {
...
+  font-weight: bolder;
+}
+
+.hidden {
+  display: none;
+}
+
+.visible {
+  display: visible;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: this is not valid CSS, it isn't doing anything

File: css/styles.css [hunk 3]
```diff
@@ -57,3 +57,56 @@ textarea {
...
+
+.entries-style {
+  font-size: 1.8rem;
+  font-weight: bolder;
+}
+
+.hidden {
+  display: none;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: THis is part of the starter code I gave you in `layout.css`

File: js/main.js [hunk 4]
```diff
@@ -21,7 +21,103 @@ function submitEntry(event) {
...
+      </div>
+    </div>
+  </div>
+</li> */
+
+function renderJournalEntry(entry) {
+  var li = document.createElement('li');
+
+  var container = document.createElement('div');
+  container.setAttribute('class', 'container');
```
Comment (codingcodymiller) [inline js/main.js]: Why an individual `container` for each `li`? That seems unnecessary. You usually only need one `container` at the most external layer of a section. For this project, I usually recommend making the `main` element the `container`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (danielycho) [reply css/styles.css]: fixed.

File: css/styles.css (hunk 2 above)
Comment (danielycho) [reply css/styles.css]: fixed

File: css/styles.css (hunk 3 above)
Comment (danielycho) [reply css/styles.css]: fixed

File: js/main.js (hunk 4 above)
Comment (danielycho) [reply js/main.js]: Fixed

**Review by codingcodymiller — APPROVED** [review]
